app/assets/stylesheets/components/banner.scss

```
@@ -142,3 +142,56 @@
   }
 
 }
+
+.banner-dashboard {
+
+  $baseline-shift: 5px;
+
+  display: block; // for browsers that don't support flexbox
+  display: flex;
+  align-items: baseline;
+  flex-wrap: wrap;
+  padding: $gutter-two-thirds 0;
+  border-top: 1px solid $border-colour;
+  border-bottom: 1px solid $border-colour;
+  margin-bottom: $gutter;
+  text-decoration: none;
+
+  &:focus {
+    border: none;
```

Border size need to be consistent with the non-focused state otherwise it changes height when focused
```suggestion
    border-top: 1px solid transparent;
    border-bottom: 1px solid transparent;    
```